packages/crayons-i18n/i18n/en-US.json

```diff
@@ -168,5 +168,10 @@
     "placeholder": {
       "search": "Type to search"
     }
+  },
+  "countryPhone": {
```

Comment:
@Vignesh-Manogar-E3433 : I have created messages in i18 but when calling on my constants.js file as below, these are not getting binded with translations, I'm ending up with the key itself. 
```

export const messages = {
  TOO_SHORT: TranslationController.t('countryPhone.tooShort'),
  INVALID_COUNTRY: TranslationController.t('countryPhone.invalidCountry'),
  TOO_LONG: TranslationController.t('countryPhone.tooLong'),
};

```

Reply:
Deleted this message as we are doing any validation inside the component.